gradle.properties

```diff
@@ -1,4 +1,4 @@
 group = run.smt.ktest
-version = 1.1.0
+version = 1.1.1
```

Comment:
If you want this fix to be released quickly as bugfix you need to create PR in appropriate versioned develop, not just to develop. Also version should be changed in release, not in PR